src/mpd.c

```diff
@@ -3589,6 +3593,30 @@ mpd_command_password(struct evbuffer *evbuf, int argc, char **argv, char **errms
   return ACK_ERROR_PASSWORD;
 }
 
+static int
+mpd_command_binarylimit(struct evbuffer *evbuf, int argc, char **argv, char **errmsg, struct mpd_client_ctx *ctx)
+{
+  unsigned int size;
+
+  if (argc <= 1)
```

Comment:
This just logs an error message, no error is returned to the client. Shouldn't it be implemented like in e.g. [mpd_command_consume](https://github.com/owntone/owntone-server/blob/master/src/mpd.c#L1132)? I also notice that the other handlers don't check argc. Instead min_argc is set in the handler struct (you are setting it to -1, but I think it should be 2).

Reply:
right I copied probably not the best example function